cli.js

```diff
@@ -60,8 +55,8 @@ const scriptArgs = process.argv.slice(3)
   .join(' ')
 
 searchForScript(__dirname, scriptPath)
-  .then(
-    foundPath => executeScript(foundPath, scriptArgs),
-    triedPaths => throwSearchFail(scriptPath, triedPaths)
-  )
-  .catch(() => process.exit(1))
+  .then(foundPath => executeScript(foundPath, scriptArgs))
+  .catch(e => {
+    console.error(e.toString())
+    process.exit(1)
+  })
```

Comment:
Right, however I don't think that is comparing apples to apples.  See https://github.com/petkaantonov/bluebird/wiki/Promise-anti-patterns#the-thensuccess-fail-anti-pattern

>`.catch` is specified for built-in Javascript promises and is "sugar" for `.then(null, function(){})`.